assets/js/components/products-control/index.js

```diff
@@ -2,110 +2,91 @@
  * External dependencies
  */
 import { __, _n, sprintf } from '@wordpress/i18n';
-import { Component, Fragment } from '@wordpress/element';
-import { debounce, find } from 'lodash';
-import PropTypes from 'prop-types';
+import { Fragment } from '@wordpress/element';
 import { SearchListControl } from '@woocommerce/components';
+import PropTypes from 'prop-types';
 
 /**
  * Internal dependencies
  */
-import { isLargeCatalog, getProducts } from '../utils';
-
-class ProductsControl extends Component {
-	constructor() {
-		super( ...arguments );
-		this.state = {
-			list: [],
-			loading: true,
-		};
-
-		this.debouncedOnSearch = debounce( this.onSearch.bind( this ), 400 );
-	}
-
-	componentDidMount() {
-		const { selected } = this.props;
-
-		getProducts( { selected } )
-			.then( ( list ) => {
-				this.setState( { list, loading: false } );
-			} )
-			.catch( () => {
-				this.setState( { list: [], loading: false } );
-			} );
-	}
-
-	componentWillUnmount() {
-		this.debouncedOnSearch.cancel();
-	}
-
-	onSearch( search ) {
-		const { selected } = this.props;
-		getProducts( { selected, search } )
-			.then( ( list ) => {
-				this.setState( { list, loading: false } );
-			} )
-			.catch( () => {
-				this.setState( { list: [], loading: false } );
-			} );
-	}
-
-	render() {
-		const { list, loading } = this.state;
-		const { onChange, selected } = this.props;
+import { withSearchedProducts } from '../../hocs/with-searched-products';
```

Comment:
You can import it directly from `index.js`, right?
```suggestion
import { withSearchedProducts } from '../../hocs';
```

Reply:
Yup, good catch!  Fixed in 34cd8ed (another place I could've just committed your suggestion).